gradle.properties

```diff
@@ -0,0 +1,3 @@
+guavaVersion = 27.1-android
```

Comment:
Could the "dot notation" be used here? So `guava.version`?

Reply:
I believe it could, but I've never seen it done for "custom" variables that are referenced in `build.gradle`. I believe it would then have to be accessed as `project.properties['guava.version']`